state-chain/custom-rpc/src/lib.rs

```diff
@@ -1350,6 +1353,7 @@ mod test {
 						]),
 					),
 				]),
+				network_fee: Permill::from_percent(0),
```

Comment:
A non-zero value would be better so we know how the Permill is represented in json (ie. would 10% be 0.1? or 100 (permill)?